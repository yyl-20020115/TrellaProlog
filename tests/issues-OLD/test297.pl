main :-
	A=A-A,
	term_variables([A,_X,_Y,_Z], L),
	write(L), nl.

:- initialization(main).
