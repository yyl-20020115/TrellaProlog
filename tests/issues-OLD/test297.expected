[_4,_5,_6]
